countString.js

```diff
@@ -0,0 +1,21 @@
+// Count each letter in string and return number of times letter appears in string
+const countLetters = function (string) {
+  const results = {};
+
+  for (const s of string) {
+    if (results[s]) {
+      results[s] += 1;
+    } else {
+      results[s] = 1;
+    }
+  }
+  return results;
+};
+
+const words = "First Hackathon Woop Woop".split(" ").join("");
+
+const results1 = countLetters(words);
+
+console.log(results1);
```

Comment:
It appears that you are testing your code. It could also be extracted and create a new file inside de test folder to test this functionality.